packages/cli/test/fixtures/discover/index.js

```diff
@@ -0,0 +1,23 @@
+const fs = require('fs');
+
+exports.SANDBOX_FILES = [
+  {
+    path: '',
+    file: 'test-schema-defs.json',
+    content: fs.readFileSync(
+      require.resolve('./models/test-schema-defs.json.txt'),
+    ),
+  },
+  {
+    path: 'dist/src/datasources',
+    file: 'mem.datasource.json',
+    content: fs.readFileSync(
+      require.resolve('./models/mem.datasource.json.txt'),
```

Comment:
Ditto, can we use `mem.datasource.json`?

Reply:
Done